.drone.yml

```diff
@@ -1,16 +1,40 @@
 ---
 kind: pipeline
-name: default
+name: testing
 
 platform:
   os: linux
   arch: amd64
 
 steps:
+- name: vet
+  pull: always
+  image: golang:1.11
+  commands:
+  - go vet ./...
+  environment:
+    GO111MODULE: on
+  volumes:
+  - name: gopath
+    path: /go
+
 - name: test
+  pull: always
   image: golang:1.11
   commands:
-  - go test -v --cover ./...
+  - go test -cover ./...
+  environment:
+    GO111MODULE: on
```

Comment:
this is not required in Go 1.11 if the build is happening outside the GOPATH

Reply:
Just wanted to make sure it's really set as on the plugins or other repos, but if you prefer I can drop it.